app/src/main/java/org/openimis/imispolicies/Enquire.java

```diff
@@ -462,6 +447,28 @@ private String getDataFromDb(String chfid) {
         return result;
     }
 
+    protected AlertDialog showDialog(String msg, DialogInterface.OnClickListener okCallback, DialogInterface.OnClickListener cancelCallback) {
+        AlertDialog.Builder builder = new AlertDialog.Builder(this)
+                .setMessage(msg)
+                .setCancelable(false);
+
+        if (okCallback != null) {
+            builder.setPositiveButton(R.string.Ok, okCallback);
+        } else {
+            builder.setPositiveButton(R.string.Ok, ((dialog, which) -> dialog.cancel()));
+        }
+
+        if (cancelCallback != null) {
+            builder.setNegativeButton(R.string.Cancel, cancelCallback);
+        }
+
+        return builder.show();
+    }
+
+    protected AlertDialog showDialog(String msg) {
+        return showDialog(msg, null, null);
+    }
+
```

Comment:
This should not be required, you can get these features by changing base class of `Enquire` class to `ImisActivity` and fixing some incompabilities